### PR TITLE
[OSDOCS-2252] Update OKE vs OCP features table

### PR DESCRIPTION
**Pull request by maxwelldb**
This PR addresses [OSDOCS-2252](https://issues.redhat.com/browse/OSDOCS-2252). 

Preview: https://deploy-preview-33769--osdocs.netlify.app/openshift-enterprise/latest/welcome/oke_about

### CODE REVIEW

File: welcome/oke_about.adoc [hunk 1]
```diff
@@ -216,278 +216,120 @@ deployments.
...
+| {rh-openstack-first} Kuryr Integration | Included | Included | N/A
+| Ingress Cluster-wide Firewall | Included | Included | N/A
+| Egress Pod and Namespace Granular Control | Included | Included | N/A
+| Ingress Non-Standard Ports | Included | Included | N/A
+| Multus and Available Multus Plugins | Included | Included | N/A
+| Network Policies | Included | Included | N/A
+| IPv6 Single and Dual Stack | Included | Included | N/A
+| CNI Plugin ISV Compatibility | Included | Included | N/A
+| CSI Plugin ISV Compatibility | Included | Included | N/A
+| RHT and IBM middleware à la carte purchases (not included in {product-title} or {oke}) | Included | Included | N/A
```
Comment (maxwelldb) [inline welcome/oke_about.adoc:266]: I'm confused about what this row is supposed to indicate. Am I misreading, or are these purchases both included and not for both OCP and OKE?

File: welcome/oke_about.adoc [hunk 2]
```diff
@@ -216,278 +216,120 @@ deployments.
...
+| ISV or Partner Operator and Container Compatibility (not included in {product-title} or {oke}) | Included | Included | N/A
```
Comment (maxwelldb) [inline welcome/oke_about.adoc:267]: Same question as in L266.

File: welcome/oke_about.adoc [hunk 3]
```diff
@@ -216,278 +216,120 @@ deployments.
...
+| Embedded OperatorHub | Included | Included | N/A
+| Embedded Marketplace | Included | Included | N/A
+| Quay Compatibility (not included) | Included | Included | N/A
```
Comment (maxwelldb) [inline welcome/oke_about.adoc:270]: `(not included) | Included | Included` reads confusingly to me, again.

File: welcome/oke_about.adoc [hunk 4]
```diff
@@ -216,278 +216,123 @@ deployments.
...
+| Red Hat Build of Quarkus | Not included | Included | N/A
+| Kourier Ingress Controller | Not included | Included | N/A
+| RHT Middleware Bundles Sub Compatibility (not included in {product-title}) | Not included | Included | N/A
+| IBM Cloud Pak Sub Compatibility (not included in {product-title}) | Not included | Included | N/A
+| OpenShift Do (`odo`) | Not included | Included | N/A
+| Source to Image and Tekton Builders | Not included | Included | N/A
+| OpenShift Serverless FaaS | Not included | Included | N/A
+| IDE Integrations | Not included | Included | N/A
+| Windows Machine Config Operator | Community Windows Machine Config Operator included - no subscription required | Red Hat Windows Machine Config Operator included - Requires separate subscription | Windows Machine Config Operator
+| Red Hat Quay | Not Included - Requires separate subscription | Not Included - Requires separate subscription | Quay Operator
```
Comment (maxwelldb) [inline welcome/oke_about.adoc:310]: Could these all just be "Subscription required"?

**Comment by maxwelldb** [comment]
@tkatarki Any comments on this so far? I did have a few comments/questions/suggestions.

**Comment by tkatarki** [comment]
LGTM

**Review by adellape — APPROVED** [review]

**Comment by maxwelldb** [comment]
@vikram-redhat Merging now that this is approved.

**Pull request merged**